resources/startup.sh

```diff
@@ -142,9 +148,18 @@ if [ "$(doguctl config successfulInitialConfiguration)" != "true" ]; then
   exportNexusPassword
 
   export NEXUS_SERVER="http://localhost:8081/nexus"
```

Comment:
This should be part of the installDefaultDockerRegistry function. Maybe you could also use the -s parameter of nexus-claim